Open Judge System/Workers/OJS.Workers.ExecutionStrategies/JavaSpringAndHibernateProjectExecutionStrategy.cs

```diff
@@ -0,0 +1,486 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using System.Xml;
+    using Checkers;
```

Comment:
Separate using statements from different main namespaces and write the full namespace path.